### PR TITLE
Adding must gather jenkins job

### PR DESCRIPTION
**Pull request by paigerube14**
Successfully got must-gather to run with increased pod memory and cpu sizing. Will want to add to sub jobs to gather data on failures 

@qiliRedHat @svetsa-rh @skordas PTAL and let me know which jobs we feel this would be the most beneficial 

https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/paige-e2e-multibranch/job/must-gather/21/console

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+            # Get ENV VARS Supplied by the user to this job and store in .env_override
+            echo "$ENV_VARS" > .env_override
+
+            # Export those env vars so they could be used by CI Job
+            set -a && source .env_override && set +a
+            mkdir -p ~/.kube
+            cp $WORKSPACE/flexy-artifacts/workdir/install-dir/auth/kubeconfig ~/.kube/config
+            ls
+            echo -e "upgrade post check find err, collect must-gather....\n"
+            schedulable_masters=(`oc get node | grep master |egrep -v 'Ready,SchedulingDisabled|NotReady'|awk '{print $1'}`)
```
Comment (svetsa-rh) [inline Jenkinsfile:114]: So, exclude master nodes in "Ready" state as well?

**Review by svetsa-rh — COMMENTED** [review]
Nightly tests should collect must gather if cerberus fails, but I think adding a checkbox to allow user to run "Must Gather" under Upgrade, Loaded upgrade and Regression jobs after Cerberus check will be handy and useful, unless by default we want to collect must gather every time Cerberus check fails.

Upgrade job currently doesn't have cerberus check (unless you use loaded upgrade).

File: Jenkinsfile (hunk 1 above)
Comment (paigerube14) [reply Jenkinsfile:114]: I believe that if no masters are schedulable the must gather isn't able to create a pod/run the gather at all.

**Comment by paigerube14** [comment]
@svetsa-rh what are your thoughts on adding this as a checkbox to cerberus and the upgrade job to get a must gather if the check fails?

File: Jenkinsfile (hunk 1 above)
Comment (svetsa-rh) [reply Jenkinsfile]: I misread earlier.

**Comment by svetsa-rh** [comment]
Paige, I am fine with what we discussed:
- Ability to collect must-gather when cerberus fails during loaded upgrade, regression and upgrade jobs
- Ability to perform cerberus check and above in upgrade job

/lgtm

**Comment by skordas** [comment]
I got my own must-gather :) https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/paige-e2e-multibranch/job/must-gather/24/ 
/lgtm

**Pull request merged**